### ISSUE TITLE
Hello, I'd like to ask, does every video require preprocessing before editing?

### ISSUE THREAD

**Issue by a869072989** [omerbt/TokenFlow#28]
This is an excellent project, but I'm curious, does each video need preprocessing before editing? Thank you very much.

**Comment by rakesh-reddy95** [comment]
yes, because you need latents to edit the video. Or you can initiate with random noise which may not reconstruct properly. You can generate the latents ( preprocess) once and edit the video n times with different prompts.
